src/pages/home/report/ReportActionCompose.js

```diff
@@ -589,6 +589,7 @@ class ReportActionCompose extends React.Component {
                                                 </Tooltip>
                                             </View>
                                             <PopoverMenu
+                                                animationInTiming={100}
```

Comment:
@Uros787 For this one, I feel we can just create a new `CONST.ANIMATION_IN_TIMING: 100` and replace it here?

Reply:
minor thing anyways, ill go with your proposal but if you change your mind ill create another commit